snomed/com.b2international.snowowl.snomed.model/src/com/b2international/snowowl/snomed/SnomedConstants.java

```diff
@@ -375,6 +375,8 @@ private Concepts() { }
 		public static final String PRODUCT_TERM_PLURAL = "69701000132106";
 		public static final String SEARCH_TERM = "9221000132108";
 		public static final String SHORT_NAME = "9211000132103";
+		public static final String THIRTY_CHAR_SHORT_NAME = "9241000132102";
+		public static final String SIXTY_CHAR_SHORT_NAME = "9251000132104";
```

Comment:
If description length validation is going to be dynamic based on values of the Description Type reference set then these constants are not needed at all. Keeping customer specific constants away from the snow-owl repo is something we are trying to achieve in general. I will keep this PR open until the necessary changes are made. Let me know if you don't have any more changes in this repo and I will close this.